cypress/e2e/shared/legends.test.ts

```diff
@@ -540,7 +549,9 @@ describe.skip('Legends', () => {
           cy.getByTestID('page-header').click().type(cellName)
 
           // Save it to a dashboard
+          cy.get('.veo-contents').click() // click out of inline editor
           cy.getByTestID('save-cell--button').click()
```

Comment:
This is needed because the save-cell button isn't properly selected by Cypress unless you click out of the inline editor. This is the same fix we've employed on other tests here.